src/main/java/me/botsko/prism/database/sql/SqlSelectQueryBuilder.java

```diff
@@ -281,7 +281,7 @@ private void keywordCondition() {
         // Keyword(s)
         final String keyword = parameters.getKeyword();
         if (keyword != null) {
-            addCondition("ex.data LIKE '%" + keyword + "%'");
+            addCondition("MATCH(ex.data) AGAINST('" + keyword + "')");
```

Comment:
I would add a wildcard operator and the `IN BOOLEAN MODE` modifier, giving
`addCondition("MATCH(ex.data) AGAINST('" + keyword + "*'  IN BOOLEAN MODE)");`

This is technically not the same as the old version; to get that, you'd need `('*" + keyword + "*'  IN BOOLEAN MODE)")` but at that point, you obviate the advantage of a full text search (and, in fact, FTS does not support this).